### DISCUSSION TITLE
Getting a missing cookie error on running up (yarn dev)

### DISCUSSION THREAD

**Discussion by VaraSampath** [profilecity/vidur#41] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/730e7be8-4c69-41ee-a6b5-017139adf0e7)

**Comment by raiss15** [comment, marked as answer]
You can follow the below steps:

- Download docker and pgadmin

- Fort and clone the repo
- Run command in VS
          * npm i -g yarn        
          * install yarn
          * docker compose up
         
- In PgAdmin connect the database
        After connecting with the Vidur database right right-click and  "go to Create script" write the command "UPDATE users SET is_admin = true;" to get into the admin dashboard.  http://localhost:3001/admin
- In docker, you can also create the dev env

Try out these steps.
